recipes/igwn-lldd-lsmp/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "igwn-lldd-lsmp" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "088604e1ec8f467b985cce0c213714362ed8aa79ccb66fe13ef0d4519d963a28" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

No need for a jinja variable that will be used only once.

```suggestion
  sha256: 088604e1ec8f467b985cce0c213714362ed8aa79ccb66fe13ef0d4519d963a28
```